pandas/core/generic.py

```diff
@@ -5292,46 +5295,120 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
                scipy < 0.18
 
         axis : {0, 1}, default 0
-            * 0: fill column-by-column
-            * 1: fill row-by-row
-        limit : int, default None.
+            Axis to interpolate along.
+
+            * 0: Fill column-by-column.
+            * 1: Fill row-by-row.
+        limit : int, default None
             Maximum number of consecutive NaNs to fill. Must be greater than 0.
+        inplace : bool, default False
+            Update the data in place if possible.
         limit_direction : {'forward', 'backward', 'both'}, default 'forward'
+            If limit is specified, consecutive NaNs will be filled in this
+            direction.
         limit_area : {'inside', 'outside'}, default None
-            * None: (default) no fill restriction
-            * 'inside' Only fill NaNs surrounded by valid values (interpolate).
-            * 'outside' Only fill NaNs outside valid values (extrapolate).
+            If limit is specified, consecutive NaNs will be filled with this
+            restriction.
+
+            * None: No fill restriction (default).
+            * 'inside': Only fill NaNs surrounded by valid values
+              (interpolate).
+            * 'outside': Only fill NaNs outside valid values (extrapolate).
+
             .. versionadded:: 0.21.0
 
-            If limit is specified, consecutive NaNs will be filled in this
-            direction.
-        inplace : bool, default False
-            Update the NDFrame in place if possible.
         downcast : optional, 'infer' or None, defaults to None
             Downcast dtypes if possible.
-        kwargs : keyword arguments to pass on to the interpolating function.
+        kwargs
+            Keyword arguments to pass on to the interpolating function.
 
         Returns
         -------
-        Series or DataFrame of same shape interpolated at the NaNs
+        Series or DataFrame
+            Same-shape object interpolated at the NaN values
 
         See Also
         --------
-        reindex, replace, fillna
+        replace : replace a value
+        fillna : fill missing values
 
         Examples
         --------
 
-        Filling in NaNs
+        Filling in NaNs in a Series via linear interpolation.
 
-        >>> s = pd.Series([0, 1, np.nan, 3])
-        >>> s.interpolate()
-        0    0
-        1    1
-        2    2
-        3    3
+        >>> ser = pd.Series([0, 1, np.nan, 3])
+        >>> ser.interpolate()
+        0    0.0
+        1    1.0
+        2    2.0
+        3    3.0
         dtype: float64
 
+        Filling in NaNs in a Series by padding, but filling at most two
+        consecutive NaN at a time.
+
+        >>> ser = pd.Series([np.nan, "single_one", np.nan,
+        ...                  "fill_two_more", np.nan, np.nan, np.nan,
+        ...                  4.71, np.nan])
+        >>> ser
```

Comment:
To save space you don't need to print the `Series` here - should be straightforward based off the constructor directly above it

Reply:
I did not include the print for the very small series example (it was straightforward to see), but I'd like to keep this longer one if that's alright - it was encouraged so the differences can be spotted easier.